### PR TITLE
Added recipe file for yamilovs sms bundle

### PR DESCRIPTION
**Pull request by yamilovs**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+yamilovs_sms:
+    providers:
+        # You must to define at least one provider configuration
+        # See more info at README.md file
```
[the remark this answers is not in the document]
Comment (OskarStark) [reply yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml]: is providers key mandatory?

[the remark this answers is not in the document]
Comment (yamilovs) [reply yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml]: Yeah, it's strongly necessary. You can't send a message without at least one message provider

**Comment by Nyholm** [comment]
Thank you for this PR. 
It looks like we just are adding an empty config file in this recipe. Could we add something that makes this recipe more value? 

Flex is smart enough to figure out how to install this bundle without a recipe.

File: yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nyholm) [reply yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml]: Could you give a link to https://github.com/yamilovs/SmsBundle#configuration

**Comment by yamilovs** [comment]
Hey! Thanks for your replies!
Ok, i'll create a post-install message with configuration link, and add same link in config comments.
I'll do it in few days.

**Comment by yamilovs** [comment]
@Nyholm, hello!
I added `post-install.txt` file with configuration reference and also added a link in the documentation.

Thanks!

File: yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yamilovs) [reply yamilovs/sms-bundle/1.0/config/packages/yamilovs_sms.yaml]: Yeah, done in https://github.com/symfony/recipes-contrib/pull/253/commits/ffd0c02465450e0b21feeaef78aac07e7b11e913

File: yamilovs/sms-bundle/1.0/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                                           </>
+<bg=blue;fg=white> Getting started using </><bg=blue;fg=white;options=bold>yamilovs/sms-bundle</><bg=blue;fg=white> </>
```
Comment (Nyholm) [inline yamilovs/sms-bundle/1.0/post-install.txt]: We are trying to remove the use of too many colors. Could you please remove this?

Comment (Nyholm) [reply yamilovs/sms-bundle/1.0/post-install.txt]: Also, the extra markup you are using `<bg=blue;fg=white;options=bold>` does not do anything... 

screen shot 2018-04-03 at 11 02 15


Just keep the markup in the beginning of the line and Im very happy with this PR.

Comment (yamilovs) [reply yamilovs/sms-bundle/1.0/post-install.txt]: Done at https://github.com/symfony/recipes-contrib/pull/253/commits/645011b994a2ab60f9b7bb48a2f6f6bb7fdc2119

**Pull request merged**